src/main/java/frc/robot/Constants.java

```diff
@@ -39,6 +39,33 @@ public static final class EndEffectorConstants {
         public static final double kPlaceCubeOutput = -0.8; //not yet decided 
     }
 
+    public static final class ArmConstants{
+        public static final double stageOne_kP = 0.5; //undecided
+        public static final double stageOne_kI = 0; //undecided
+        public static final double stageOne_kD = 0; //undecided
+
+        public static final double stageTwo_kP = 0.5; //undecided
```

Comment:
This might be way too high for rev encoder

Reply:
doesnt really matter, when we go to tune we will find out